src/main/java/trinsdar/gravisuit/items/tools/ItemToolAdvancedChainsaw.java

```diff
@@ -87,7 +92,10 @@ public float getDestroySpeed(ItemStack stack, IBlockState state) {
 
     @Override
     public boolean itemInteractionForEntity(ItemStack stack, EntityPlayer playerIn, EntityLivingBase entity, EnumHand hand) {
-        return Ic2Items.chainSaw.getItem().itemInteractionForEntity(stack, playerIn, entity, hand);
+    	if (this.getDamage(stack) == 0) {
```

Comment:
Please chane this to a simple nbt boolean; i'm trying to use as little metadata as possible, the gravitool being the only one using metadata as i needed the texture to change.